draft-dekater-scion-dataplane.md

```diff
@@ -447,8 +449,18 @@ The SCION common header has the following packet format:
 | other          |      |                      | Unassigned     |
 {: #table-3 title="Allocations of type values to length values"}
 
-- `RSV`: These bits are currently reserved for future use.
+A service address designates a set of endpoint addresses rather than a singular one. A packet addressed to a service is redirected to any one endpoint-addresses that is known to be part of the set.
+
+Current known values are:
 
+| Hexadecimal value | Short Name | Description            |
+|-------------------+------------+------------------------|
+| 0x00000001        | DS         | Discovery Service      |
+| 0x00000002        | CS         | Control Service        |
+| 0x0000ffff        | None       | Reserved invalid value |
+{: #table-4 title="Known Service values"}
+
+- `RSV`: These bits are currently reserved for future use.
```

Comment:
Would it make more sense to place the overall service addresses table one section below, at the end of parargaph `## Address Header` ? 

I am saying this because service addresses are special values of `DstHostAddr` when `DT/DL`=1/0, this field is in the address header, not in the common header

Reply:
Done.